contrib/examples/prometheus/README.md

```diff
@@ -0,0 +1,87 @@
+# Prometheus Metrics
+
+Service Catalog maintains metrics through the Prometheus client library and
+exposes them through the Prometheus http adapter at the Controller's /metrics
+endpoint. These metrics can be accessed directly via HTTP GET or more commonly
+scraped by the Prometheus monitoring application which persists metrics and
+facilitates analysis through a Web UI and powerful query language.
+
+Many metrics are not created and exposed until Service Catalog performs
+operations which would impact the metrics.  IE if you have no Service Brokers
+defined, there will be no metrics around class or plan count and likely no OSB
+Client operation metrics.  So before proceeding, it's recommended you have a
+Broker defined and created a Service Instance.
+
+To view the raw metrics:
+
+```
+# setup a port forward so we can curl against Controller Manager
+
+$ kubectl get pods -l app=catalog-catalog-controller-manager -n catalog -o name | \
+    sed 's/^.*\///' | xargs -I{} kubectl port-forward {} -n catalog 8089:8080 &
+
+$ curl -s http://localhost:8089/metrics  | grep servicecatalog
+Handling connection for 8089
+# HELP servicecatalog_broker_service_class_count Number of services classes by Broker.
+# TYPE servicecatalog_broker_service_class_count gauge
+servicecatalog_broker_service_class_count{broker="ups-broker"} 1
+# HELP servicecatalog_broker_service_plan_count Number of services classes by Broker.
+# TYPE servicecatalog_broker_service_plan_count gauge
+servicecatalog_broker_service_plan_count{broker="ups-broker"} 2
+# HELP servicecatalog_osb_request_count Cumulative number of HTTP requests from the OSB Client to the specified Service Broker grouped by broker name, broker method, and response status.
+# TYPE servicecatalog_osb_request_count counter
+servicecatalog_osb_request_count{broker="ups-broker",method="Bind",status="2xx"} 41
+servicecatalog_osb_request_count{broker="ups-broker",method="GetCatalog",status="2xx"} 1
+servicecatalog_osb_request_count{broker="ups-broker",method="ProvisionInstance",status="2xx"} 2
+```
+
+Alternatively, and the more common approach to utlizing metrics, deploy
+Prometheus.  [This YAML](prometheus.yml) creates a Prometheus instance
+preconfigured to gather Kubernetes platform and node metrics.  If you deploy the
+Service Catalog Controller Manager via Helm with the optional
+`enablePrometheusScrape` parameter set to true (see
```

Comment:
We should probably write the exact `helm set` here in addition to the general descriptive guideline.